### DISCUSSION TITLE
GSoC contribution

### DISCUSSION THREAD

**Discussion by 0xsambit** [json-schema-org/community#831] [General]
Hello devs, I was looking forward to join the community here for GSoC contribution next year. The slack channel link has expired. 

Looking forward to get started in open source contribution in this organization. Need guidance.

Thank you.

**Comment by benjagm** [comment]
Hi @0xsambit . Please use this link to join Slack: https://join.slack.com/t/json-schema/shared_invite/zt-2ued3v79g-Tk_aI32ZdW~ST0EWpGBwNQ

**Comment by 0xsambit** [comment]
Thanks @benjagm . **Would you also specify where to talk about the next GSoC.**
